src/components/VideoPlayer.tsx

```diff
@@ -22,23 +22,23 @@ export const VideoPlayer = ({
     }
     const handleKeyPress = (event: any) => {
       switch (event.code) {
-      case 'Space': // Space bar for play/pause
-        if (player.paused()) {
```

Comment:
Same question here

Reply:
these are just noise, no real change done from my side

Reply:
> Same question here

This change fixes the bug of videoplayer controls not working on full screens.